roles/edpm_nova/tasks/configure.yml

```diff
@@ -1,5 +1,17 @@
 ---
 
+- name: Create sync config dirs
+  tags:
+    - configure
+    - nova
+  become: true
+  ansible.builtin.file:
+    path: "{{ lookup('ansible.builtin.env', 'HOME') }}/staging"
+    state: "directory"
+    mode: "0755"
```

Comment:
mode shoudl eirther by 766 or you should set the owner to the current user

but since this is in your home directory you should be able to just remove become:true instaed

Reply:
thanks, done!